src/api/routers/participants/participantsUsers.ts

```diff
@@ -30,6 +31,7 @@ export async function handleInviteUserToParticipant(req: UserParticipantRequest,
   try {
     const { participant, user } = req;
     const userPartial = userInvitationSchema.parse(req.body);
+    const userRoleIdData = UpdateUserRoleIdSchema.parse(req.body);
```

Comment:
I'm using a separate schema here because the userRoleId is not part of the User table, whereas everything inside `userData` is. That means the `userData` can be used as a "partial user" to do inserts/updates directly on the User entity, while including `userRoleId` in that schema would break that.